inotify.go

```diff
@@ -162,6 +162,20 @@ func (w *Watcher) Remove(name string) error {
 	return nil
 }
 
+// WatchList returns the directories and files that are being monitered.
+func (w *Watcher) WatchList() []string {
```

Comment:
Be careful with adding new APIs that don't exist on other platforms. We want to keep a uniform API to the underlying systems.

@mattn